### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by ankita-sharma2019**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Solutions/Armorblox/Analytic Rules/ArmorbloxNeedsReviewAlert.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+id: 322d4765-be6b-4868-9e3f-138a4f339dd6
+name: Armorblox Needs Review Alert
+description: |
+  'This rule generates an alert on fo the armorblox incident where remediation action is "Needs Review".'
```
Comment (urajat) [inline Solutions/Armorblox/Analytic Rules/ArmorbloxNeedsReviewAlert.yaml]:
```suggestion
  'This rule generates an alert for an Armorblox incident where the remediation action is "Needs Review".'
```

File: Solutions/Armorblox/Playbooks/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Needs-Review-Incident-Email-Notification
+**Author:** Armorblox
+
+This playbook will send an Email notification when a new incident is opened in Azure Sentinel.
```
Comment (urajat) [inline Solutions/Armorblox/Playbooks/README.md]:
```suggestion
This playbook will send an email notification when a new incident is created in Azure Sentinel.
```

File: Solutions/Armorblox/Playbooks/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Pre-requisites
+An O365 account to be used to send email notification (The user account will be used in O365 connector (Send an email).)
```
Comment (urajat) [inline Solutions/Armorblox/Playbooks/README.md]: What is the `(Send an email)` bit?

File: Solutions/Armorblox/Playbooks/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Deployment                                                                                         
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Farmorblox%2FAzure-Sentinel%2Fxoriant%2FSolutions%2FArmorblox%2FPlaybooks%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Farmorblox%2FAzure-Sentinel%2xoriant%2FSolutions%2FArmorblox%2FPlaybooks%2Fazuredeploy.json)
+ 
+
+## Post-deployment
+
+### Configure connections
+Once this Playbooks template is deployed, you will need to go into the Logic App, edit it and click on each of the steps that require an authenticated connection to your tenant and complete the connection process. These steps will have an exclamation point showing that the connection needs to be completed. Make sure to also open the "For each" step which also contains a step that requires an authenticated connection.
```
Comment (urajat) [inline Solutions/Armorblox/Playbooks/README.md]:
```suggestion
Once this Playbooks template is deployed, you will need to go into the Logic App, edit it and click on each of the steps that require an authenticated connection to your tenant and complete the connection process. These steps will have an exclamation mark showing that the connection needs to be completed. Make sure to also open the "For each" step which also contains a step that requires an authenticated connection.
```

File: Solutions/Armorblox/Playbooks/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Note:  Email sent with this playbook will be from user that creates connection!
```
Comment (urajat) [inline Solutions/Armorblox/Playbooks/README.md]:
```suggestion
Note:  Emails sent with this playbook will be from the user that creates the connection.
```

File: Solutions/Armorblox/Playbooks/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+### Attach the playbook
+After deployment, attach this playbook to the `Armorblox Needs Review` analytic rule so it runs when the alert is created.</br>
+Note: Playbook is disabled by default. Please enable it before assigning to the Automation rule!
```
Comment (urajat) [inline Solutions/Armorblox/Playbooks/README.md]:
```suggestion
Note: Playbook is disabled by default. Please enable it before assigning to the Automation rule.
```

**Review by urajat — CHANGES_REQUESTED** [review]

File: Solutions/Armorblox/Playbooks/README.md (hunk 3 above)
Comment (ankita-sharma2019) [reply Solutions/Armorblox/Playbooks/README.md]: It represents the step 2 in the playbook. o365 is the brand and the feature used from the brand is `send an email`

Comment (ankita-sharma2019) [reply Solutions/Armorblox/Playbooks/README.md]: Should I remove it `(Send an email)` ?

Comment (urajat) [reply Solutions/Armorblox/Playbooks/README.md]: That's fine. I think you can change this line to `An O365 account to be used to send email notification. The user account will be used in O365 connector (Send an email).`

**Review by urajat — APPROVED** [review]

File: Solutions/Armorblox/Playbooks/README.md (hunk 3 above)
Comment (ankita-sharma2019) [reply Solutions/Armorblox/Playbooks/README.md]: Done

**Pull request merged**